pq-crypto/bike_r2/defs.h

```diff
@@ -23,7 +23,7 @@
 ////////////////////////////////////////////
 //             Basic defs
 ///////////////////////////////////////////
-#define FUNC_PREFIX BIKE1_L1_R2
+#define FUNC_PREFIX BIKE1_L1_R1
```

Comment:
Is this correct? This file is `bike_r2/defs.h`.

Reply:
Bad merge, fixed.